seeds/farm-dev/add-farm-groups.js

```diff
@@ -302,7 +294,12 @@ function generateFakeFarmData (index) {
     name: `${faker.name.firstName()} ${faker.name.lastName()}`,
     farm_leadership_experience: Math.random() > 0.85 ? null : Math.random() * 19 + 1,
     area: Math.random() > 0.6 ? null : generateFakeGeometry(),
-    location,
+    location_address_line1: location.address_line1,
+    location_address_line2: null,
+    location_locality: location.locality,
+    location_country_code: location.country_code,
```

Comment:
huh, i am actually working on a PR for the location obfuscation that changes our database column name for country to country_code and adds a new country column for storing full country text so we can display that